web3js-1.0/src/rpc.ts

```diff
@@ -33,12 +33,16 @@ export async function getStakeActivation(
     })(),
   ]);
 
-  const { effective, activating, deactivating } =
+  const { effective, activating, deactivating } = stakeAccount.stake ?
     getStakeActivatingAndDeactivating(
       stakeAccount.stake.delegation,
       BigInt(epochInfo.epoch),
       stakeHistory
-    );
+    ) : {
+      effective: BigInt(0),
+      activating: BigInt(0),
+      deactivating: BigInt(0),
+    };
```

Comment:
Rather than zeroing these, I prefer to keep this code consistent with the web3.js v2 version which only looks at the discriminant. Unfortunately, the discriminant is incorrect in the web3.js v1 version, so we can go with this for now